Services/Dashboard/Access/class.DashboardAccess.php

```diff
@@ -46,17 +46,17 @@ protected function getSettingsRefId(): int
                 AND object_data.type = %s
                 AND object_reference.ref_id = tree.child
                 AND object_reference.obj_id = object_data.obj_id',
-                array('integer', 'text'),
-                array(SYSTEM_FOLDER_ID, 'dshs')
+                ['integer', 'text'],
```

Comment:
Can you change this as well? As you are changing this line anyway?
```suggestion
                [ilDBConstants::T_INTEGER, ilDBConstants::T_TEXT],
```

Reply:
Im aware of this.
I will tackle this issue in another PR to keep the overview of the refactoring changes.